### PR TITLE
feat: switch to use new pipeline trigger by prow in branch 8.1

### PR DESCRIPTION
**Pull request by purelind**
Swtich to use new pipeline trigger by prow in tikv & tiflash branch release-8.1

### CODE REVIEW

**Review by glorv — APPROVED** [review]

File: jenkins/jobs/ci/tiflash/ghpr_build.groovy [hunk 1]
```diff
@@ -56,6 +56,7 @@ pipelineJob('tiflash-ghpr-build') {
                     orgslist('pingcap')
                     blackListTargetBranches {
                         ghprbBranch { branch('master') }
+                        ghprbBranch { branch('release-8.1') }
```
Comment (wuhuizuo) [inline jenkins/jobs/ci/tiflash/ghpr_build.groovy:59]: description: disable the old CI job.

File: prow-jobs/pingcap/tiflash/release-8.1-presubmits.yaml [hunk 2]
```diff
@@ -9,22 +9,20 @@ presubmits:
...
       decorate: false # need add this.
-      # skip_if_only_changed: *skip_if_only_changed
-      always_run: false
-      context: wip/pull-unit-test
-      skip_report: true
-      optional: true
+      skip_if_only_changed: *skip_if_only_changed
+      context: pull-unit-test
+      skip_report: false
+      optional: false
```
Comment (wuhuizuo) [inline prow-jobs/pingcap/tiflash/release-8.1-presubmits.yaml:15]: description: GA for the refactored jobs.

**Comment by purelind** [comment]
## tiflash
image
image
## tikv
image

**Comment by wuhuizuo** [comment]
/lgtm
/hold

wait for more LGTMs

**Review by Lloyd-Pottiger — APPROVED** [review]

**Pull request merged**